client/js/jsonedit.js

```diff
@@ -554,6 +554,21 @@ const jeCommands = [
       jeSetAndSelect(w.get('width'));
     }
   },
+  {
+    id: 'je_moveLimit',
+    name: 'move limits',
+    context: '^[^ ]* ↦ moveLimit',
+    call: async function() {
+      const w = widgets.get(jeStateNow.id);
+      jeStateNow.moveLimit.push({
```

Comment:
I don't think you want limit to be an array - especially since the code only ever looks at the first index.